### PR TITLE
Notes on Setting up Spark on Kubernetes (with Magnum) on Openstack

### PR DESCRIPTION
**Pull request by stvoutsin**
Pull request contains two files, first describes the process and commands run to firstly setup Kubernetes on Openstack with Magnum, and then to create a Spark cluster with kubectl and custom Spark images based on the project here: https://github.com/devshlabs/spark-kubernetes
Second file contains some notes on scaling up the cluster, and running sample Spark jobs

### CODE REVIEW

File: notes/stv/20200408-Spark-on-K8s.txt [hunk 1]
```diff
@@ -288,28 +288,6 @@ Key name [stv-master]
 | flavor_id           | 20061eba-9e88-494c-95a3-41ed77721244                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 +---------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-## [Admin Node]  Create storage Class
```
Comment (Zarquan) [inline notes/stv/20200408-Spark-on-K8s.txt]: Does this mean the nodes don't have any storage on them ?

Comment (stvoutsin) [reply notes/stv/20200408-Spark-on-K8s.txt]: I noticed that the Storage class create command is run a few lines after, so this was probably a mistake

File: notes/stv/20200408-Spark-on-K8s.txt [hunk 2]
```diff
@@ -461,17 +463,17 @@ git clone https://github.com/devshlabs/spark-kubernetes
...
 
-nano kubernetes/spark-worker-controller.yaml
+nano spark-kubernetes/kubernetes/spark-worker-controller.yaml
 ..
 
     spec:
       containers:
         - name: spark-worker
-          image: stvoutsin/spark:2.4.4
+       stvoutsin/spark:2.4.4   image:
```
Comment (Zarquan) [inline notes/stv/20200408-Spark-on-K8s.txt:472]: Is this a typo ?

    image: stvoutsin/spark:2.4.4

makes more sense than

    stvoutsin/spark:2.4.4   image:

Comment (stvoutsin) [reply notes/stv/20200408-Spark-on-K8s.txt:472]: Yep thats a typo, should be -> image: stvoutsin/spark:2.4.4

**Pull request merged**